### DISCUSSION TITLE
[Catalog.waitForReady():876] wait catalog to be ready. FE type: UNKNOWN. is ready: false

### DISCUSSION THREAD

**Discussion by kkfnui** [apache/doris#11460] [RFC]
今天在扩容新的 FE 节点。 遇到操作失败的问题 —— 最后新 FE 节点无法添加到集群中。 错误日志：
```
[Catalog.waitForReady():876] wait catalog to be ready. FE type: UNKNOWN. is ready: false
```

在峰哥的指导下解决了问题。官方的运维可[操作文档](https://doris.apache.org/zh-CN/docs/admin-manual/cluster-management/elastic-expansion?_highlight=%E6%89%A9%E5%AE%B9#%E5%A2%9E%E5%8A%A0-fe-%E8%8A%82%E7%82%B9)可能有问题。根据我实际的操作得出的结论：

**错误的顺序**：

1. 在系统注册：ALTER SYSTEM ADD FOLLOWER "follower_host:edit_log_port";
2. 启动新节点：./bin/start_fe.sh --helper leader_fe_host:edit_log_port --daemon

**正确的顺序**：

1. 启动新节点：./bin/start_fe.sh --helper leader_fe_host:edit_log_port --daemon
2. 在系统注册：ALTER SYSTEM ADD FOLLOWER "follower_host:edit_log_port";

> 环境信息： 
> - 旧 fe 节点： 1.0.0 版本
> - 扩容 fe 节点: 1.1.1 版本

**Comment by kpfly** [comment]
Could you submit a pull request to correct this document, thanks
